stonesoup/regulariser/particle.py

```diff
@@ -32,6 +32,14 @@ class MCMCRegulariser(Regulariser):
 
     transition_model: TransitionModel = Property(doc="Transition model used for prediction",
                                                  default=None)
+    constraint_func: Callable = Property(
+        default=None,
+        doc="Callable, user defined function for applying "
+            "constraints to particle states. This is done by reverting "
+            "that are moved to a state outside of the defined constraints "
```

Comment:
```suggestion
            "constraints to particle states. This is done by reverting particles "
            "that are moved to a state outside of the defined constraints "
```

Reply:
Well spotted. Fixed and improved clarity in latest commit.